src/utils/loadParseJson.js

```diff
@@ -0,0 +1,39 @@
+//import RNFS from 'react-native-fs';
+const loadParseJson = (url) => {

```

Comment:
nitpick: In the future,  when we actually implement the network call to fetch the text we can rename this function name that describes more of what it does.